### DISCUSSION TITLE
BUG: inaccurate GitHub mobile feed

### DISCUSSION THREAD

**Discussion by foxyseta** [community/community#20890] [Mobile] [answered]
My GitHub Mobile feed (the one you see in the "Discover" tab, under the "Trending Repositories" and "Awesome Lists" button) features false information. It says a user published a discussion in a certain repository when, in fact, they just starred it. My desktop "For you" feed, instead correctly states that the repository was starred.

I'm currently on GitHub Mobile v1.73.0-beta (10077).

**Comment by Iron-Ham** [comment]
Hello @FoxySeta! 

Can I get some more information from you? Is this occurring on iOS or Android? Is it possible for you to share a screenshot of what you're experiencing?

**Reply by foxyseta** [reply]
Thanks for your answer! I'm on Android. Here are two screenshots, comparing my web app and mobile app experiences. The accurate one is the the web app's.

**Reply by foxyseta** [reply]
![Screenshot_20220720-195012](https://user-images.githubusercontent.com/12249077/180049468-c256a354-d5fc-4d76-8a6e-10e3592aca3f.png)
![Screenshot_20220720-194932](https://user-images.githubusercontent.com/12249077/180049472-abf218db-5d8d-40ed-a6d8-8242e73d9030.png)

**Reply by foxyseta** [reply]
bump

**Comment by MrSarno** [comment]
> I'm currently on GitHub Mobile v1.73.0-beta (10077).

Are you able to reproduce this on the latest non-beta release?

**Reply by foxyseta** [reply]
Which version code (e. g. 10078, not 1.74.0) is that?

**Reply by foxyseta** [reply]
bump

**Comment by aconsuegra** [comment, marked as answer]
Hi @FoxySeta, thanks for the bug report! We fixed the issue and will be rolling out a new version with the fix this week.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
